file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
After poking around on Windows for a while, it turns out we already have a `share` directory alongside `bin` and `lib` in `C:\Program Files\Puppet Labs\Puppet\puppet`, which for some reason isn't listed here. I agree with Michael, it makes sense to put `locale` in there for consistency.